### ISSUE TITLE
Potential FEPs

### ISSUE THREAD

**Issue by luzpaz** [FreeCAD/FreeCAD-Enhancement-Proposals#2]
- [ ]  import freecad on all platforms without `sys.path` modifications [discussion](https://forum.freecadweb.org/viewtopic.php?f=4&t=40768&p=352170#p352157)

**Comment by looooo** [comment]
other topics:
- freecad command-line interface (standardization)
- standardization of freecad namespace-packages
- release numbering scheme (mentioned here: https://github.com/FreeCAD/FreeCAD-Enhancement-Proposals/pull/1#pullrequestreview-317347238)

**Comment by looooo** [comment]
- handling of line endings: https://forum.freecadweb.org/viewtopic.php?f=17&t=41117&p=353191#p353191

**Comment by luzpaz** [comment]
I'll start adding some in the next few days.

**Comment by sgrogan** [comment]
> I'll start adding some in the next few days.

IMHO there should be less, not more FEP's
I think we should use this tool for issues that impact multiple developers, where we need consensus and a coordinated plan. I don't think this should turn into a wish list.

Again IMHO I think we should concentrate on quality not quantity.

I would vote for the name-space issue, as this is a big change that needs a good plan.
And as a packager, the versioning issue, that should be easier, but requires coordination, and some sort of "standard" so it can be flowed down to 3rd party packagers and not just the FreeCAD team.

Just my 2 cents.

**Comment by Vanuan** [comment]
@sgrogan I don't think there would be many even without the restriction of what would be a FEP. There are only 2 proposals since this initiative is started which means nobody wants to put an effort even if it was a wishlist.

I think that first we should migrate all the [projects](https://wiki.freecadweb.org/Category:Roadmap) into this format.

**Comment by sgrogan** [comment]
@Vanuan I can't disagree that this effort didn't take hold. The idea was to create "projects" that could be agreed upon so that teams would hopefully be created pulling in a common direction. Hasn't worked so far.

The wiki model you link is no more successful, at least now. It's woefully out of date, most developers mentioned aren't even active in the project anymore. The link shows a summary of what had been discussed in the forum. The community was smaller then and consensus was easier.

I don't know the answer and wouldn't reject anything out of hand. I just don't know how to make it work. I think in your proposal most projects would simply be empty. Even for the WB's where we have maintainers I don't think there is a true roadmap.

That doesn't mean we should give up. Sometimes we need champion's. i.e. name space WB's. @looooo has done 99% of the work and has been a consistent advocate. Some developers have begun to convert there WB's.

TLDR; I don't know

**Comment by Vanuan** [comment]
> standardization of freecad namespace-packages

Is this FEP02? https://github.com/FreeCAD/FreeCAD-Enhancement-Proposals/wiki/FEP02-Namespace-packages-and-python-imports
